docroot/themes/humsci/humsci_basic/src/scss/components/_chosen.scss

```diff
@@ -170,7 +170,6 @@
     font-size: hb-calculate-rems(16px);
     line-height: 1;
     height: auto;
-    max-height: $hb-input-min-height;
```

Comment:
We want to keep this property to make the input used with selecting `Quarter` options the same height as the other inputs. Without a `max-height` property, it's a little taller.
![Screen Shot 2020-05-07 at 3 21 22 PM](https://user-images.githubusercontent.com/12678977/81336091-ae1e8d80-9076-11ea-8b5e-9065175ed332.png)